packages/lodestar-cli/src/cmds/account/cmds/validator/index.ts

```diff
@@ -0,0 +1,18 @@
+import {CommandBuilder} from "yargs";
+import {accountValidatorOptions, IAccountValidatorOptions} from "./options";
+import * as create from "./create";
+import * as deposit from "./deposit";
+
+export const command = "validator <command>";
+
+export const description = "Provides commands for managing Eth2 validators.";
+
+export const builder: CommandBuilder<{}, IAccountValidatorOptions> = (yargs) => {
+  return yargs
+    .options(accountValidatorOptions)
+    .command(create)
+    .command(deposit);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export function handler(): void {}
```

Comment:
I think we should use "dev" command style export as it's much more typesafe in terms what are you suppose to export and with which type:
https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar-cli/src/cmds/dev/index.ts

cc @wemeetagain

Reply:
I'm not sure about that. Using this exports style the main weakness is misspelling the named exports. However, besides that, yargs does a great job at infering the option and handlers types and make sure they are consistent. Also using the `CommandModule` approach forced the author to cast the module entirely to `any` in `lodestar-cli/src/index.ts`:

https://github.com/ChainSafe/lodestar/blob/8524b8086320997bc91251111af45782b4da60cf/packages/lodestar-cli/src/index.ts#L14-L15

Reply:
Agree! I've converted all commands to the style you suggest except for the third level child commands such as `account validator create`, which I get type errors on their parent builder that the options do not match.